docs/src/components/LandingPage.module.css

```
@@ -0,0 +1,563 @@
+  --landing-line-strong: oklch(0.76 0 0);
+  --landing-surface: oklch(0.985 0 0);
+  --landing-surface-raised: oklch(1 0 0);
+  --landing-muted: oklch(0.43 0 0);
+  --landing-quiet: oklch(0.55 0 0);
+  position: relative;
+  width: 100%;
+  overflow: clip;
+  background: var(--background);
+}
+
+:global(.dark) .page {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)